workers/linux/relay_worker.py

```diff
@@ -107,7 +114,8 @@ def handle_message(self, message):
                         self.relay_active.clear()
                     Logger.log(
                         LOG_LEVEL["info"],
-                        'Switch Relay \033[1;36m{0}\033[0;0m state to \033[1;36m{1}\033[0;0m'.format(
+                        'Switch Relay \033[1;36m{0}\033[0;0m state \
```

Comment:
I know this might be too much to ask but maybe we should consider using the constants https://github.com/mudpi/mudpi-core/blob/feature/constants.py#L266-L271 or if they are not defined then define them and use them.